keps/sig-node/4328-node-label-manager/README.md

```diff
@@ -0,0 +1,442 @@
+<!--
```

Comment:
it's a good idea to remove the guiding comments once you addressed them to reduce the visual noise and make the review a tad easier